src/components/views/FilesDropdown.tsx

```diff
@@ -83,6 +87,63 @@ const isFileValidForFigure = (
   }
 }
 
+const getFigures = (doc: ProsemirrorNode) => {
+  const _figures = new Map<string, string>()
```

Comment:
why underscore is used here for variables naming?

Reply:
_ is removed